src/screens/Notification.js

```diff
@@ -361,6 +361,30 @@ background: {
             <Example defaultValue>"background-front"</Example>
           </PropertyValue>
         </Property>
+        <Property name="notification.critical.message.color">
```

Comment:
It looks like we're missing the kind specific options for each status 

`notification.critical.global.message.color`
`notification.critical.toast.message.color`